tests/integration_tests/test_hyperopt_ray.py

```diff
@@ -47,9 +47,9 @@
         "utterance.fc_layers": {
             "space": "choice",
             "categories": [
-                [{"fc_size": 512}, {"fc_size": 256}],
-                [{"fc_size": 512}],
-                [{"fc_size": 256}],
+                [{"output_size": 512}, {"output_size": 256}],
+                [{"output_size": 512}],
+                [{"output_size": 256}],
```

Comment:
these values is likely too big, let's decrease it for faster tests

Reply:
Done